crates/typst-library/src/visualize/gradient.rs

```diff
@@ -706,6 +706,46 @@ impl Gradient {
         }
     }
 
+    /// Returns the radius of this gradient.
```

Comment:
```suggestion
    /// Returns the center of this gradient.
```

Reply:
Would also be nice to specify that this returns `none` for linear gradients. (We can update the documentation of `angle` similarly.)